### PR TITLE
[match] Clean up match/sync_code_signing docs

### PR DESCRIPTION
**Pull request by janpio**
I read through the whole document and cleaned up a few repetitions, awkward sentences etc.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/docs/sync_code_signing.md [hunk 1]
```diff
@@ -58,29 +60,26 @@ Before starting to use _match_, make sure to read the [codesigning.guide](https:
...
+You'll be asked if you want to store your code signing identities inside a **Git repo**, or on **Google Cloud**.
 
 #### Git Storage
 
 Use Git Storage to store all code signing identities in a private git repo, owned and operated by you. The files will be encrypted using OpenSSL.
 
-First, enter the URL to your Git repo. This can be either a `https://` or a `git` URL. (If your machine is currently using SSH to authenticate with GitHub, you'll want to use a `git` URL, otherwise you may see an authentication error when you attempt to use match.) `fastlane match init` won't read or modify your certificates or profiles yet, and also won't validate your git URL.
+First, enter the URL to your private (!) Git repo (You can create one for free on e.g. [GitHub](https://github.com/new) or [BitBucket](https://bitbucket.org/repo/create)). The URL you enter can be either a `https://` or a `git` URL. (If your machine is currently using SSH to authenticate with GitHub, you'll want to use a `git` URL, otherwise you may see an authentication error when you attempt to use match.) `fastlane match init` won't read or modify your certificates or profiles yet, and also won't validate your git URL.
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/docs/sync_code_signing.md]: What is the ` (!) ` there for? 🤔

File: fastlane/lib/fastlane/actions/docs/sync_code_signing.md [hunk 2]
```diff
@@ -189,17 +193,17 @@ _match_ will reuse certificates and will create separate provisioning profiles f
 
 When running _match_ for the first time on a new machine, it will ask you for the passphrase for the Git repository. This is an additional layer of security: each of the files will be encrypted using `openssl`. Make sure to remember the password, as you'll need it when you run match on a different machine.
 
-To set the passphrase to decrypt your profiles using an environment variable, use `MATCH_PASSWORD`.
+To set the passphrase to decrypt your profiles using an environment variable, and avoid the prompt, use `MATCH_PASSWORD`.
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/docs/sync_code_signing.md]:
```suggestion
To set the passphrase to decrypt your profiles using an environment variable (and avoid the prompt) use `MATCH_PASSWORD`.
```

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Looks good! One small question and one small change request

File: fastlane/lib/fastlane/actions/docs/sync_code_signing.md (hunk 1 above)
Comment (janpio) [reply fastlane/lib/fastlane/actions/docs/sync_code_signing.md]: highlight that the repo should be private

**Review by joshdholtz — APPROVED** [review]
Looks good to me 💪 🔥

**Pull request merged**